environments/eck-ror/kind-cluster/ror/es.yml

```diff
@@ -17,6 +17,10 @@ spec:
                 runAsUser: 0
                 runAsGroup: 0
               env:
+                // For ROR_ES_VERSION >= 1.64.0
```

Comment:
I'm not sure what is the best way of adding that flag. The env variable is renamed in the ROR 1.64.0. But the new ROR version accepts also the old name in a backwards compatible way. So this change is not strictly required (is recommended)

Reply:
It's ok for now. We will remove the old flag in the future. We run e2e tests for the latest release (master branch) and pre-builds (develop) only.

BTW, please bump the pre-builds versions [here](https://github.com/beshu-tech/readonlyrest-e2e-tests/blob/83bfcbf3de32050e5c1a7d57db9d9cd060ca5314/.github/workflows/trigger-e2e-tests.yml#L52) when the pipelines ([es](https://dev.azure.com/beshu-tech/ReadonlyREST%20for%20Elasticsearch/_build/results?buildId=7811&view=results) and [kbn](https://dev.azure.com/beshu-tech/ReadonlyREST%20Kibana/_build/results?buildId=7812&view=results)) publish docker images